TestApexAssignment.cls

```diff
@@ -0,0 +1,49 @@
+@isTest(seeAllData=true)
+public class TestApexAssignment {
+
+    @isTest static void positiveTestGetAllStudentsOfClass(){
```

Comment:
Keep method name in way like testMethodFor_Scenerio_ExpectedOutPut()
For this case it should be like testGetAllStudent_withActualStudentId_ExpectedThreeStudents